metrics/opentelemetry-metrics/src/main/scala/sttp/tapir/server/metrics/opentelemetry/OpenTelemetryMetrics.scala

```diff
@@ -36,6 +36,33 @@ case class OpenTelemetryMetrics[F[_]](meter: Meter, metrics: List[Metric[F, _]])
 
 object OpenTelemetryMetrics {
 
+  /** Default labels for OpenTelemetry-compliant metrics, as recommended here:
+    * https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-server
+    *
+    *   - `http.request.method` - HTTP request method (e.g., GET, POST).
+    *   - `path` - The request path or route template.
+    *   - `http.response.status_code` - HTTP response status code (200, 404, etc.).
+    */
+  lazy val OpenTelemetryAttributes: MetricLabels = MetricLabels(
+    forRequest = List(
+      "http.request.method" -> { case (_, req) => req.method.method },
+      "url.scheme" -> { case (_, req) => req.uri.scheme.getOrElse("unknown") },
+      "path" -> { case (ep, _) => ep.showPathTemplate(showQueryParam = None) }
+    ),
+    forResponse = List(
+      "http.response.status_code" -> {
+        case Right(r) => r.code.code.toString
+        // Default to 500 for exceptions
+        case Left(_) => "500"
+      },
+      "error.type" -> {
+        case Left(ex) => ex.getClass.getSimpleName
+        case Right(_) => ""
```

Comment:
I think if the response is a `Right` we shouldn't add any attributes

Reply:
@adamw The scala compiler is hungry to handle `Right` case?
WDYT the better way what you are thinking of?

Reply:
@adam

Reply:
@adamw It's getting a bit tricky making  "error.type"  to  Option[String]. In that case, if we make  forResponse then h  http.response.status_code could go None

Reply:
yes, if an attribute is `None` we don't include it, if it's a `Some` we do